### PR TITLE
docs: Add deployment and GitHub configuration guide

### LINKED ISSUE #125: How to install the webhook

### PR DESCRIPTION
**Pull request by orestes**
Resolves #125

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,17 +17,57 @@ A webhook to forward GitHub events to StreamLabs Alerts and Twitch Chat.
...
+- [ ] Your repository (and/or user/organisation) on GitHub/GitLab is configured to notify your
+      webhook
+
+You can deploy this webhook in different ways
+
+### Deploy to Heroku
+
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
 
 ## Configuration
```
Comment (SantiMA10) [inline README.md:35]: Why is this configuration header here?

File: README.md [hunk 2]
```diff
@@ -17,17 +17,57 @@ A webhook to forward GitHub events to StreamLabs Alerts and Twitch Chat.
...
+   ```
+2. Install the dependencies with yarn (or alternative you can use `npm`)
+
+   ```
+   yarn install
+   ```
+
+   ```
+
+   ```
```
Comment (SantiMA10) [inline README.md:55]: Extra code block?

File: README.md [hunk 3]
```diff
@@ -45,6 +85,27 @@ We make use of the following environment variables:
 
 ### For sponsorships
```
Comment (SantiMA10) [inline README.md:86]: Why is the title here without any content?

File: README.md [hunk 4]
```diff
@@ -45,6 +85,27 @@ We make use of the following environment variables:
...
 
+### GitHub Configuration
+
+#### Repositories
```
Comment (SantiMA10) [inline README.md:90]: Should this title be `For repositories`?

**Comment by orestes** [comment]
@SantiMA10 I just fixed the order issues and updated the variables. LMK

**Pull request merged**